plugins.xml

```diff
@@ -2761,6 +2761,26 @@
 			<description>A minor update to support changes to the default theme in OJS 3.3.</description>
 		</release>
 	</plugin>
+	<plugin category="themes" product="material">
+		<name locale="en_US">Material Theme</name>
+		<homepage>https://github.com/madi-nuralin/material.git</homepage>
+		<summary locale="en_US">Material Theme (MDBootstrap based).</summary>
+		<description locale="en_US"><![CDATA[<p>Material Theme (MDBootstrap based), an adaptive stylesheet for both fronted and backend templates.</p>]]></description>
+		<maintainer>
+			<name>Madi Nuralin</name>
+			<institution>International Information Technology University</institution>
+			<email>nmdbzk@gmail.com</email>
+		</maintainer>
+		<release date="2021-08-25" version="1.0.0.0" md5="711f0ee8d746e11c2ae72fb12b7c89a3">
+			<package>https://github.com/madi-nuralin/material/releases/download/v1_0_0-0/material-v1_0_0-0.tar.gz</package>
```

Comment:
@madi-nuralin, the plugin would need to be packaged in a directory called `material`, not `pluginName`. Could you rebuild the package, update the md5sum, and let me know here to have a second look?